### ISSUE TITLE
`Add Me!` It is necessary to create a multifunctional, modular simulation of the QSP library

### ISSUE THREAD

**Issue by l3ger0j** [l3ger0j/Questopia#39]
For further development and transition of the project to the "Stable, suitable for everyday use" stage, it is necessary to create an imitation of the QSP library in the C++ or Rust programming language, which will include the capabilities of both QSP 5.7.0 authored by Byte, and a later variation of QSP 5.7.0 authored by Sonnix, as well as the current QSP 5.8.0 from Byte.

Why do I think so? Because here it will not be possible to do with minor changes in the library code due to the fact that this can lead to a situation that is the reverse of the one in which the project is currently located (that is, some games will work correctly, and some will not).

**In summary, for further development, the project needs to modify the QSP library in order to ensure support for the launch of many existing and existing game projects.**

**Comment by l3ger0j** [comment]
Games that are not working now (updated list):

1. Clean State. Reason: using a slightly different syntax inherent QSP 5.7.0 from Sonnix;
